tests/testthat/test-gather_images.R

```diff
@@ -0,0 +1,10 @@
+test_that("gather_images works", {
+  
+  species <- c("Mus_musculus","Pan_troglodytes","Homo_sapiens")
+  res <- rphylopic::gather_images(species=species)
+  
+  testthat::expect_equal(species, res$species)
+  testthat::expect_true(
+    all(c("species","uid","namebankID","string","picid") %in% colnames(res))
```

Comment:
this line also fails. did you run it?

Reply:
Fixed. Seems I had another version of this function in my namespace when i originally tested it.